### ISSUE TITLE
[FEATURE REQUEST] Feature Picker

### ISSUE THREAD

**Issue by 0xlne** [TrashboxBobylev/Community-Pixel-Dungeon#3]
As this mod is an amalgam of player requests, preferences are quite individual.
It would be useful to be able to select individual (features), a bit like we are able to select specific challenges.

**Comment by TrashboxBobylev** [comment]
Implemented in-dev.

**Comment by 0xlne** [comment]
> Implemented in-dev.

That's mighty appreciated. Thank you; is the dev branch private? I can only see master.

**Comment by TrashboxBobylev** [comment]
Yeah, it's only accessible to me right now. Don't have enough features in it for release yet.
